### PR TITLE
refactor(Core/Server): Move Banner into separate file

### PR DESCRIPTION
**Pull request by Kitzunu**
## Changes Proposed:
- Remove duplicate banner and throw it into its own file  

## Tests Performed:

- Tested on Windows

## How to Test the Changes:

1. Run CMake
2. Run Worldserver & Authserver

## Target Branch(es):

- [x] Master

https://github.com/azerothcore/azerothcore-wotlk/pull/4692 needs to be merged first

 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: src/server/authserver/Main.cpp [hunk 1]
```diff
@@ -81,27 +82,18 @@ extern int main(int argc, char** argv)
...
-    sLog->outString("                                ██║     ██║   ██║██╔══██╗██╔══╝");
-    sLog->outString("                                ╚██████╗╚██████╔╝██║  ██║███████╗");
-    sLog->outString("                                 ╚═════╝ ╚═════╝ ╚═╝  ╚═╝╚══════╝\n");
-
-    sLog->outString("     AzerothCore 3.3.5a  -  www.azerothcore.org\n");
-
-    sLog->outString("Using configuration file %s.", configFile.c_str());
-
-    sLog->outDetail("%s (Library: %s)", OPENSSL_VERSION_TEXT, SSLeay_version(SSLEAY_VERSION));
+        acore::Banner::Show("authserver",
```
Comment (FrancescoBorzi) [inline src/server/authserver/Main.cpp:85]: indentation

File: src/common/Banner.cpp [hunk 2]
```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2016+     AzerothCore <www.azerothcore.org>, released under GNU GPL v2 license, you may redistribute it and/or modify it under version 2 of the License, or (at your option), any later version.
+ * Copyright (C) 2008-2016 TrinityCore <http://www.trinitycore.org/>
```
Comment (FrancescoBorzi) [inline src/common/Banner.cpp:3]: I think we can remove TC from here

Comment (Kitzunu) [reply src/common/Banner.cpp:3]: only reason I put it there is because this is basically a copy-paste from TC but with our banner

Comment (Kitzunu) [reply src/common/Banner.cpp:3]: thoguhts?

Comment (Kitzunu) [reply src/common/Banner.cpp:3]: ^ Me trying to figure out what that means O.o

**Comment by Winfidonarleyan** [comment]
Better after #4692

**Comment by Winfidonarleyan** [comment]
With the current logging system PR bad

**Comment by Kitzunu** [comment]
we could just put it in your PR if you prefer

**Comment by Winfidonarleyan** [comment]
I think need separated pr for minimum changes

**Comment by Kitzunu** [comment]
Then we just wait til it's merged

**Comment by Winfidonarleyan** [comment]
Can help test my logging pr?

**Comment by Kitzunu** [comment]
I just built your PR. What do you need tested?

File: src/common/Banner.cpp (hunk 2 above)
Comment (Helias) [reply src/common/Banner.cpp:3]: for me it's the same, we can keep it or remove
this project is clearly based on TC, so it does not matter I think

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by Kitzunu** [comment]
I will rework this to new logging system when that PR is merged

**Comment by Kitzunu** [comment]
Can you put this PR as draft @FrancescoBorzi 
Im not allowed to convert an already open PR to it

File: src/common/Banner.cpp [hunk 3]
```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2016+     AzerothCore <www.azerothcore.org>, released under GNU GPL v2 license, you may redistribute it and/or modify it under version 2 of the License, or (at your option), any later version.
```
Comment (Winfidonarleyan) [inline src/common/Banner.cpp:2]: v3

**Comment by Winfidonarleyan** [comment]
Looks like, need add screenshoot with this

**Review by Winfidonarleyan — APPROVED** [review]

**Comment by Kitzunu** [comment]
image

File: src/server/authserver/Main.cpp [hunk 4]
```diff
@@ -12,6 +12,7 @@
 * authentication server
 */
 
+#include "Banner.h"
```
Comment (Winfidonarleyan) [inline src/server/authserver/Main.cpp:15]: for better need after `Common.h`

**Review by Winfidonarleyan — APPROVED** [review]

**Pull request merged**